lib/stripe_mock/request_handlers/tokens.rb

```diff
@@ -34,16 +32,13 @@ def create_token(route, method_url, params, headers)
           params[:card][:last4] = params[:card][:number][-4,4]
           customer_card = params[:card]
         elsif params[:bank_account].is_a?(String)
-          customer = assert_existence :customer, cus_id, customers[stripe_account][cus_id]
-
           # params[:bank_account] is an id; grab it from the db
           bank_account = verify_bank_account(customer, params[:bank_account])
           assert_existence :bank_account, params[:bank_account], bank_account
         elsif params[:bank_account]
           # params[:card] is a hash of cc info; "Sanitize" the card number
           bank_account = params[:bank_account]
         else
-          customer = assert_existence :customer, cus_id, customers[stripe_account][cus_id] || customers[Stripe.api_key][cus_id]
```

Comment:
The only case that handled correctly the source cloning, but if you want to clone a bank account or card, it won't work.